client/src/partials/general_ledger/general_ledger.html

```diff
@@ -30,6 +30,7 @@
           ui-grid-auto-resize
           ui-grid-resize-columns
           ui-grid-move-columns
+          ui-grid-cellNav
```

Comment:
Does the general ledger need `cellNav`? If the services are making it depend on `cellNav`, we should redesign the services!